main/src/library/InputStream.flix

```diff
@@ -0,0 +1,89 @@
+namespace InputStream {
```

Comment:
Let us use the namespaces and put this into `Core/Io/InputStream`.

Reply:
Do you want me to move the file to match the namespace like you would in Java?

Reply:
No, I think we are going to go with our own namespaces.